alws/alembic/versions/585ab06efb32_extended_build_task_refs.py

```diff
@@ -0,0 +1,30 @@
+"""Extended build_task_refs
+
+Revision ID: 585ab06efb32
+Revises: cbd577117a13
+Create Date: 2021-11-14 10:40:24.138348
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '585ab06efb32'
+down_revision = 'cbd577117a13'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('build_task_refs', sa.Column('ref_type', sa.Integer(), nullable=True))
+    op.create_unique_constraint(None, 'repositories', ['name', 'arch', 'type', 'debug'])
```

Comment:
Add constraint name, because otherwise it will fail to be dropped in case someone needs downgrade

Reply:
Thanks, fixed